ign-sensors6.yaml

```diff
@@ -0,0 +1,41 @@
+repositories:
+  ign-cmake:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-cmake
+    version: ign-cmake2
+  ign-common:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-common
+    version: ign-common4
+  ign-math:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-math
+    version: ign-math6
+  ign-msgs:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-msgs
+    version: ign-msgs7
+  ign-plugin:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-plugin
+    version: ign-plugin1
+  ign-rendering:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-rendering
+    version: ign-rendering5
+  ign-sensors:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-sensors
+    version: main
+  ign-transport:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-transport
+    version: ign-transport10
```

Comment:
I believe we are missing `ign-tools`: https://github.com/ignitionrobotics/ign-sensors/blob/main/.github/ci/packages.apt#L7

Reply:
Nice catch, added to all supported `ign-sensors` versions: ad6f5df

Reply:
Should we also add it to `sensors3` (Citadel)? It looks like that's the only other supported version that's missing it after https://github.com/ignition-tooling/gazebodistro/commit/ad6f5df67e77b3ba5f3c08d97b83ecd95450d5a7.

Reply:
So many versions! Yup: 78393d6